src/Bicep.LangServer/Handlers/BicepForceModulesRestoreCommandHandler.cs

```diff
@@ -5,9 +5,6 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Azure.Deployments.Core.Entities;
-using Azure.Deployments.Core.Helpers;
-using Azure.Deployments.Core.Json;
```

Comment:
I see you're removing some extraneous `using` statements, but isn't there an analyzer rule you need to turn on somewhere to enforce it going forwards?

Reply:
It's this one `.globalconfig`:

`
# Remove unnecessary import
dotnet_diagnostic.IDE0005.severity = warning
`

Reply:
However, there are cases where this rule requires documentation in the code, thus applying the suggestion from https://github.com/dotnet/roslyn/issues/41640#issuecomment-985780130 to have the rule trigger for all files.